webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-student.js

```diff
@@ -81,7 +83,15 @@ class SakaiRubricStudent extends RubricsElement {
     return html`
       <hr class="itemSeparator" />
 
-      <h3>${this.rubric.title}</h3>
+      <h3>${this.rubric.title}
+      ${this.exportPdf=='true' ? html`
+      <span class="hidden-sm hidden-xs sr-only"><sr-lang key="export_label" /></span>
+      <a role="button" title=" ${this.rubric.title}" tabindex="0" class="linkStyle pdf fa fa-file-pdf-o" @click="${this.pdfRubric}"></a>
```

Comment:
Weird logic and using strings instead of boolean.

As Adrian mentioned do not use the "? X : Y" logic, just use Litelement templates.

Reply:
Done.